### PR TITLE
Fix: do not have newlines in log messages.

### PR DESCRIPTION
**Pull request by fisx**

### CODE REVIEW

**Comment by neongreen** [comment]
```
  test-integration/Test/Spar/APISpec.hs:342:70: 
  1) Spar.API, binding existing users to sso identities, GET /sso-initiate-bind/:idp, known IdP, running session with sso user, responds with 200 and a bind cookie
       uncaught exception: ErrorCall
       no wire cookie
       CallStack (from HasCallStack):
         error, called at test-integration/Util/Core.hs:589:27 in main:Util.Core
         loginSsoUserFirstTime, called at test-integration/Test/Spar/APISpec.hs:342:70 in main:Test.Spar.APISpec
```

File: services/gundeck/src/Gundeck/Push/Websocket.hs [hunk 1]
```diff
@@ -99,7 +99,8 @@ logBadCannons (uri, (err, prcs)) = do
     forM_ prcs $ \prc ->
         Log.warn $ logPresence prc
             ~~ Log.field "created_at" (ms $ createdAt prc)
-            ~~ Log.msg (val "WebSocket presence unreachable: " +++ (toByteString $ show (uri, err, resource prc)))
+            ~~ Log.msg (val "WebSocket presence unreachable: " +++
+                        show (uri, resource prc, show err))
```
Comment (neongreen) [inline services/gundeck/src/Gundeck/Push/Websocket.hs]: Perhaps remove the duplicate `show`?

Comment (neongreen) [reply services/gundeck/src/Gundeck/Push/Websocket.hs]: Also it might be better to report the URI and resource via `Log.field`.

Comment (neongreen) [reply services/gundeck/src/Gundeck/Push/Websocket.hs]: Oh wait, the duplicate `show` is actually what makes things work. There should be a comment about it, then.

Comment (fisx) [reply services/gundeck/src/Gundeck/Push/Websocket.hs]: Ok, I will elaborate on this in a different PR.

**Pull request merged**